src/UNL/UCBCN/Manager/Controller.php

```diff
@@ -205,6 +206,26 @@ public function getCalendars() {
         return $user->getCalendars();
     }
 
+    public function hasPermission($permission_description, $calendar_id) 
+    {
+        switch ($permission_description) {
```

Comment:
I guess I'm not sure why you need this.  Can you not just use the permission constants?